libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +66,13 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
     }
   }
+
+  ///////////////NEW////////////////
```

Comment:
You don't need to do this - I can tell what code is new. Github is fancy like that.

Reply:
Sorry about that. Those were note-to-self that I forgot to remove.